src/jit/gentree.h

```diff
@@ -2826,6 +2831,8 @@ struct GenTreeCall final : public GenTree
     { 
 #ifdef FEATURE_UNIX_AMD64_STRUCT_PASSING
         return varTypeIsStruct(gtType) && !HasRetBufArg(); 
+#elif defined(_TARGET_X86_) && !defined(LEGACY_BACKEND)
```

Comment:
On second thought, i think it would make sense to have second condition "!defined(LEGACY_BACKEND)" since it is used in flowgraph, importer, morph and lsra.cpp.